plugins/case-converter.js

```diff
@@ -0,0 +1,26 @@
+// Case converter plugin
+// Handles the conversion between the DB's snake_cased and JS's camelCased
+// Model properties automatically.
+// -----
+module.exports = function (bookshelf) {
+  'use strict';
+  var _ = require('lodash');
+
+  var Model = bookshelf.Model.extend({
+    parse: function (attrs) {
+      return _.reduce(attrs, function (memo, val, key) {
+        memo[_.camelCase(key)] = val;
+        return memo;
+      }, {});
```

Comment:
Why not do instead:

```js
parse: function(attrs) {
  return _.mapKeys(attrs, (, k) => _.camelCase(k));
}
```

Reply:
I'll convert to that method after merging this, since the original author isn't interested in doing it.